ophyd/controls/ophydobj.py

```diff
@@ -319,6 +319,17 @@ def check_value(self, value, **kwargs):
         '''
         pass
 
+    @property
+    def state(self):
+        return {}
+
+    def configure(self, state=None):
+        # does nothing; subclasses can override if configuration is possible
+        return self.state, self.state
+
+    def deconfigure(self):
+        return self.state
+
```

Comment:
I was thinking of `OphydObj` as akin to our `Base`, implementing the minimal interface of "readable" object. Maybe you were thinking of `Detector` in that role. As Tom suggests, maybe those two could be merged. Is there a reason to have a base class above the base class that all readables inherit from?